### ISSUE TITLE
Inconsistency in self-closing tag across lessons

### ISSUE THREAD

**Issue by ManonLef** [TheOdinProject/curriculum#23457]
As discussed with Eric in #contribution-suggestions;

In the [Fundamentals part 1 lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/fundamentals-part-1) we get instructed to type the basic html skeleton.
I noticed in the preview that the (self-closing) meta tag was closed with `/>`.

![Screenshot_2022-01-27_at_23 29 22](https://user-images.githubusercontent.com/81025586/151869526-a18d513f-bb57-4d52-afc3-768ca480e895.png)

It's a minor inconsistency but it threw me off a bit since we basically had to memorise the boilerplate [in this lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/html-boilerplate).
![Screenshot 2022-01-31 at 21 39 24](https://user-images.githubusercontent.com/81025586/151869905-5e36213f-a243-4a58-94af-6f25448566d0.png)

**Comment by thatblindgeye** [comment]
Would you want to take this on? There shouldn't really be any rush on it if you aren't able to get to it immediately.

Regardless, I think for whomever does take this on it would be worth searching the rest of our repo for meta tag mentions in code snippets to ensure they're consistent.

**Comment by ManonLef** [comment]
Yes, I'm willing to take this one and go through the rest to check for inconsistencies as well.

**Comment by thatblindgeye** [comment]
Awesome 🙂 You've been assigned!

**Comment by Eduardo06sp** [comment]
@ManonLef when you're done, you can double check by running 
```bash
grep -r '/>' foundations/
```
from the root directory (curriculum), it will check for closing tags inside all the lessons inside `foundations` :}
(this does not include any lesson repos of course)
